models/stg_shopify__metafield.sql

```diff
@@ -37,6 +37,8 @@ final as (
         {{ dbt_date.convert_timezone(column='cast(created_at as ' ~ dbt.type_timestamp() ~ ')', target_tz=var('shopify_timezone', "UTC"), source_tz="UTC") }} as created_at,
         {{ dbt_date.convert_timezone(column='cast(updated_at as ' ~ dbt.type_timestamp() ~ ')', target_tz=var('shopify_timezone', "UTC"), source_tz="UTC") }} as updated_at,
         {{ dbt_date.convert_timezone(column='cast(_fivetran_synced as ' ~ dbt.type_timestamp() ~ ')', target_tz=var('shopify_timezone', "UTC"), source_tz="UTC") }} as _fivetran_synced,
+        {{ dbt.concat(["namespace","'_'","key"]) }} as metafield_reference,
+        row_number() over (partition by id order by updated_at desc) = 1 as is_most_recent_record,
```

Comment:
i don't see any records where `is_most_recent_record` is false -- is this the correct partition or do we not need this?

Reply:
I noticed that there _may_ be cases where metafields can be updated so I chose to include this column and filter in the downstream macro. However, there were no updates to the data we have but I still felt it relevant to include.

Reply:
Good catch on it not being used downstream. That was the intention. I will add it into the macro.